changelog.json

```diff
@@ -1,5 +1,14 @@
 {
   "releases": {
+    "1.7.0-beta8": [
+      "[Fixed] Added accelerator and access key to \"Exit\" menu item - #6507. Thanks @AndreiMaga!",
```

Comment:
The PR this was fixed in is #7508.

Reply:
Also, should this be [Added] instead of [Fixed]???

Reply:
ohh yeah those are both good points

Reply:
actually the issue linked there is the original issue
```suggestion
      "[Added] Added accelerator and access key to \"Exit\" menu item - #6507. Thanks @AndreiMaga!",
```

Reply:
Yea, I know, figured the PR was better since we are giving someone a shoutout.